src/altinn-app-frontend/src/features/form/data/update/updateFormDataSagas.ts

```diff
@@ -91,19 +90,7 @@ function* runValidations(field: string, data: any, componentId: string | undefin
 
   let customComponentValidations: IComponentValidations = {};
   if (component.type === 'DatePicker') {
-    const fieldKey: keyof IDataModelBindings = 'simpleBinding';
-    const flagBasedMinDate = getFlagBasedDate(component.minDate as DateFlags) ?? component.minDate;
-    const flagBasedMaxDate = getFlagBasedDate(component.maxDate as DateFlags) ?? component.maxDate;
-    const datepickerValidations = validateDatepickerFormData(
-      data,
-      flagBasedMinDate,
-      flagBasedMaxDate,
-      component.format,
-      state.language.language,
-    );
-    customComponentValidations = {
-      [fieldKey]: datepickerValidations,
-    };
+    customComponentValidations = validateDatepickerFormData(data, component, state.language.language);
   }
```

Comment:
That's why I said it could be up to you to name this new(?) type of validation, as it does not include all the same validations as the function that already exists in `validation.ts` (for FileUpload*), so it must be 'something else'.

Reply:
Ahh I see